appveyor.yml

```diff
@@ -9,8 +9,8 @@ cache:
   - '%USERPROFILE%\.nuget\packages -> **\*.csproj'
 install:
   - ps: Invoke-WebRequest -Uri "https://dot.net/v1/dotnet-install.ps1" -OutFile "install-dotnet.ps1"
-  - ps: .\install-dotnet.ps1 -Version 3.1.405 -InstallDir "dotnetcli"
-  - ps: .\install-dotnet.ps1 -Version 5.0.102 -InstallDir "dotnetcli"
+  - ps: .\install-dotnet.ps1 -Version 3.1.409 -InstallDir "dotnetcli"
```

Comment:
Looks like we can rely on having 3.1.409 already installed with the update to the VS2019 base image: https://www.appveyor.com/docs/windows-images-software/#net-framework